src/metabase/premium_features/settings.clj

```diff
@@ -250,10 +250,15 @@
   "Should we allow users to embed the SDK in iframes?"
   :embedding-iframe-sdk)
 
+(define-premium-feature ^{:added "0.55.0"} enable-ai-entity-analysis?
```

Comment:
We're technically adding this in some 55.2+ version of Metabase... not sure if I should increase the point version here? Not seeing where we've done this before.

Reply:
Will do, thanks!